### ISSUE TITLE
Experiment - Visual Perspective Taking (VPT) on 3D-PC

### ISSUE THREAD

**Issue by druhinb** [druhinb/LatentInvestigation#9]
Conduct probing experiments for the VPT task on the 3D-PC dataset using both linear and MLP probes.

**Subtasks**
- Adapt training/evaluation scripts for the VPT task & dataset 
- Define appropriate metrics and select a loss function for fine-tuning
- Train and evaluate all models (DINOv2, I-JEPA, Supervised ViT) with both linear and MLP probes.
- Hydra configs for this specific experiment set.
